.babelrc

```diff
@@ -1,28 +1,26 @@
 {
   "plugins": [
-    "syntax-dynamic-import",
-    "transform-runtime",
-    "transform-object-rest-spread",
-    "transform-class-properties",
-    "transform-es2015-destructuring"
+    "@babel/plugin-syntax-dynamic-import",
+    "@babel/plugin-transform-runtime",
+    "@babel/plugin-proposal-class-properties",
+    "@babel/plugin-transform-template-literals",
+    "@babel/plugin-proposal-object-rest-spread",
+    "@babel/plugin-transform-destructuring"
   ],
   "presets": [
-    "es2015",
-    "react",
-    "stage-0"
-  ],
-  "env": {
-    "start": {
-      "presets": [
-        "react-hmre"
-      ]
-    },
-    "test": {
-      "presets": [
-        "es2015",
-        "react",
-        "stage-0"
-      ]
-    }
-  }
-}
\ No newline at end of file
+    [
+      "@babel/preset-env",
+      {
+        "useBuiltIns": "entry",
+        "targets": {
+          "node": "current",
+          "ie": "11",
+          "firefox": "62",
```

Comment:
We should target Firefox 60, since it's packed with  RHEL 7

Reply:
https://access.redhat.com/articles/2798521

Reply:
and this version will be updated with next release: we support the last version at the moment of release. Current version is 67. People can update…